NataliiaZinchenko/HW7/nzinchenko_7.3.py

```diff
@@ -0,0 +1,22 @@
+# Creating a function which calculates the number of characters included in given string
+
```

Comment:
good job
look at this version of code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result